website/docs/advanced-plugins.md

```diff
@@ -28,9 +38,75 @@ Plugins are modules which export a function that takes in the context, options a
 
 -->
 
+## Creating plugins
+
+A plugin is a module which exports a function that takes two parameters and returns an object.
+
+We need to specify for our plugin:
```

Comment:
These are not the only things a plugin is for. Only content plugins do these, the rest don't have to do so. I think saying this can be misleading for people unfamiliar to the concept of plugins.